command/git_push_test.go

```diff
@@ -0,0 +1,137 @@
+package command
+
+import (
+	"context"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/model/patch"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/rest/client"
+	"github.com/google/shlex"
+	"github.com/mongodb/jasper"
+	"github.com/stretchr/testify/assert"
```

Comment:
I suspect these tests would be improved by replacing assert with require in a lot of cases. assert's continue-on-error, often makes it harder to determine what went wrong in a test.

Reply:
replaced many of them.